test/integration/boards_test.rb

```diff
@@ -32,7 +32,7 @@ class BoardsTest < ActionDispatch::IntegrationTest
     create_card("My Card")
 
     # Then it should appear on the first list
-    within lists.first do
+    within first_list do
```

Comment:
Well, `lists.first` reaches out by index, whereas `todo_list` is the "first list", semantically speaking, hence the alias `first_list`.

I don't know, I am quite happy about it. Someday `first_list` could be something entirely different (not necessarily `todo_list`, but still would be the "first list".

So :confused: Haha